hermes-api/src/main/java/pl/allegro/tech/hermes/api/endpoints/MaintainerEndpoint.java

```diff
@@ -0,0 +1,33 @@
+package pl.allegro.tech.hermes.api.endpoints;
+
+import pl.allegro.tech.hermes.api.Maintainer;
+
+import javax.ws.rs.*;
+import javax.ws.rs.core.Response;
+import java.util.List;
+
+import static javax.ws.rs.core.MediaType.APPLICATION_JSON;
+
+@Path("maintainers")
+public interface MaintainerEndpoint {
+
+    @GET
+    @Produces(APPLICATION_JSON)
+    @Path("/sources/{source}")
+    List<Maintainer> search(@PathParam("source") String source, @QueryParam("search") String searchString);
+
+    @GET
+    @Produces(APPLICATION_JSON)
+    @Path("/sources/{source}")
+    Response searchAsResponse(@PathParam("source") String source, @QueryParam("search") String searchString);
```

Comment:
also usage of proxy client in tests is the only reason we keep these jersey interfaces